### DISCUSSION TITLE
Clue Item button in Bank similar to Quest button from quest helper.

### DISCUSSION THREAD

**Discussion by DevKatha** [runelite/runelite#14904] [Ideas]
Currently if you have a skill clue / emote clue you can search the word 'clue' in your bank and it will show only the items you need. Could a button be added, similar to that of the quest helper, where you click it and it shows the required items needed for that step. 

Could also be updated to include all clues that require items i.e Cryptic clues (gear needed to get down to legends tunnel etc etc) 

Thanks

**Comment by pwatts6060** [comment]
See similar conversation here: https://github.com/runelite/runelite/issues/14848

**Comment by Enriath** [comment]
For emote item clues, this basically already exists as the bank tag `clue`, which is auto-generated with the items for the current step. This can be made into a button using the bank tag tabs feature.
